src/api/common/EntityTypes.ts

```diff
@@ -66,4 +66,5 @@ export interface BlobElementEntity extends Entity, BlobElement {
 	_ownerGroup: null | Id
 }
 
+// FIXME ask willow about Entity
```

Comment:
what would you like to know?

Reply:
I believe `Entity` covers also `DataTransferType`s and `SomeEntity` has owner group.

We do not use `Instance` because it's a wrong type. It has `_id: Id | IdTuple` but that doesn't work because `_id` is writable *and* readable so it is invariant. You cannot unify `Element` and `ListElement` as long as `_id` is writable.